docs/product-release-notes/src/main/asciidoc/rn-known-issues-con.adoc

```diff
@@ -109,4 +109,9 @@ Serialization of KieBase/package is not supported in {PRODUCT} {PRODUCT_VERSION}
 
 .Smart Router Servers view error [https://issues.jboss.org/browse/BXMSDOC-1029[RHPAM-1029]]
 
-If a Smart Router is registered in the controller and you select one of the deployment units on the Smart Router in the *Servers* view, an error occurs. You should not manage deployment units on the Smart Router. Instead, manage the server configurations of the connected {KIE_SERVERS} directly.
\ No newline at end of file
+If a Smart Router is registered in the controller and you select one of the deployment units on the Smart Router in the *Servers* view, an error occurs. You should not manage deployment units on the Smart Router. Instead, manage the server configurations of the connected {KIE_SERVERS} directly.
```

Comment:
suggesion: "You should not" -> "Do not". "Should" is not good with IBM SG and also, this is a workaround, it's an imperative.